lib/airport.rb

```diff
@@ -1,2 +1,18 @@
+# Skapa en lista med plats för 2plan innehållande 1 plan
+# Skriv en metod som kollar hur många plan som finns ilistan - om dem är mindre än 2 är flygplatsen inte full.
```

Comment:
Metrics/LineLength: Line is too long. [110/80]<br>Style/AsciiComments: Use only ascii symbols in comments.